frontend/components/common/formatters/TimeTill.stories.js

```diff
@@ -23,4 +23,8 @@ storiesOf('Common/Formatters/TimeTill', module)
             date: new Date(Date.now() + 5000).toUTCString(),
         }),
         template: `<TimeTill :date="date" />`,
+    }))
+    .add('Invalid date', () => ({
+        components: { TimeTill },
+        template: `<TimeTill date="Invalid Date" />`,
```

Comment:
Nice that you've added a new story, but is it actually handled now? Does it show `Unknown` in case date is `undefined` or `NaN`? Otherwise I would suggest to do it in the `TimeTill.vue` – then you don't need to check it every time you use `TimeTill` – it will be handled automatically (besides the cases like with the arrow)

Reply:
I was unsure of this. Officially the component does support other types such as Strings. I don't think we currently use this in the UI at all, however it would be nice to verify this. 

Currently our fallback for the component if it can't be turned into a Date is to show the String. I think this should be fine for now, as we almost always check the value we pass to the component in the other place it is used. 

We also have different designs for how we display the "unknown" text depending on where it is. Therefore I believe it makes little sense to incorporate the error logic into the component itself.

Reply:
- [ ] Please follow the advice and set `Unknown` to be displayed as a fallback instead of `Invalid Date` or the string
  - [ ] The original input can be set as the `title` html attribute for "debugging" purposes

> We also have different designs for how we display the "unknown" text depending on where it is

If we need to display another text, there can be an additional check in the parent